include/cantera/thermo/ConstDensityTabulatedThermo.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * ConstDensityTabulatedThermo.h
+ *
+ *  Created on: 04.06.2016
+ *      Author: mmayur
+ */
+
+/**
+ *  @file ConstDensityTabulatedThermo.h
+ * Definition file for a derived class of ThermoPhase that assumes either
+ * an ideal gas or ideal solution approximation and handles
+ * variable pressure standard state methods for calculating
+ * thermodynamic properties (see \ref thermoprops and
+ * class \link Cantera::ConstDensityTabulatedThermo ConstDensityTabulatedThermo\endlink).
+ */
+/*
+ * Copyright (2005) Sandia Corporation. Under the terms of

```

Comment:
Please see one of the [existing header files](https://github.com/Cantera/cantera/blob/d7292b33459e64921074d75c92e0cd8faf37a31b/include/cantera/Edge.h#L3) for how to specify the copyright/license information.